libs/shared/devops-copilot/feature/src/lib/devops-copilot-panel/submit-vote.ts

```diff
@@ -0,0 +1,56 @@
+import { type Cluster, type Environment, type Organization, type Project } from 'qovery-typescript-axios'
+import { type AnyService } from '@qovery/domains/services/data-access'
+import { HACKATHON_API_BASE_URL } from './submit-message'
```

Comment:
This should be coming from an env variable and shouldn't be imported from the neighbor component